tests/controller_tests/test_ingest.py

```diff
@@ -29,71 +27,57 @@
     Project,
     Submission,
 )
-from core.exceptions import ValidationError
-from core.validation.exceptions import UnimplementedErrorMessageException
-from core.validation.failures.user import (
-    NonNullableConstraintFailure,
-    WrongInputFailure,
-)
 
 resources = Path(__file__).parent / "resources"
 
 
-@pytest.fixture
-def wrong_format_test_file() -> BinaryIO:
-    """An invalid text test file."""
-    with open(resources / "wrong_format_test_file.txt", "rb") as file:
-        yield file
-
-
-"""
-/ingest endpoint validation_tests
-"""
-
-
-def test_ingest_endpoint(test_client_reset, example_data_model_file, mocker):
-    """Tests that, given valid inputs, the endpoint responds successfully."""
-    load_data_mock = mocker.spy(ingest, "load_data")
-    endpoint = "/ingest"
-    response = test_client_reset.post(
-        endpoint,
-        data={
-            "excel_file": example_data_model_file,
-        },
-    )
-
-    assert load_data_mock.called
-    assert response.status_code == 200, f"{response.json}"
-    assert response.json == {
-        "detail": "Spreadsheet successfully validated and ingested",
-        "metadata": {},
-        "status": 200,
-        "title": "success",
-        "loaded": True,
-    }
-
+def example_table_data_to_be_loaded_via_populate_db():
+    """
+    Helper function that ingests example data via populate_db() in order to be loaded into the database.
 
-def test_ingest_endpoint_do_not_load(test_client_reset, example_data_model_file, mocker):
-    """Tests that, given valid inputs, the endpoint responds successfully."""
-    load_data_mock = mocker.spy(ingest, "load_data")
-    endpoint = "/ingest"
-    response = test_client_reset.post(
-        endpoint,
-        data={"excel_file": example_data_model_file, "do_load": False},
-    )
+    Enables us to test populate_db() functionality without requiring the entire ingest pipeline.
+    This function iterates through csvs in the resources folder for controller_tests and puts them into
+    a dictionary of DataFrames that is then called by populate_db().
 
-    assert not load_data_mock.called
-    assert response.status_code == 200, f"{response.json}"
-    assert response.json == {
-        "detail": "Spreadsheet successfully validated but NOT ingested",
-        "metadata": {},
-        "status": 200,
-        "title": "success",
-        "loaded": False,
-    }
+    We cannot use the csvs in the resources folder for test/ because they are in a state in which they can
+    be loaded into the database without invocation of populate_db() and will cause the function to fall over
+    if called as a parameter in it.
```

Comment:
We can't use the other CSVs because they represent an untransformed set of data right? But we need transformed data, in the structure of the data model in order to load it straight in via populate.

Not sure if this current explanation can be clearer as I didn't really understand it